macros/Plot_ResolutionCombinedPosMethod1.py

```diff
@@ -205,29 +153,38 @@ def getTH1(self, hname):
 if debugMode:
     outdir_q = myStyle.CreateFolder(outdir, "q_ResPosVsX0/")
 
-nbins = all_histoInfos[0].th2.GetXaxis().GetNbins()
 
 # Run across X-bins. ROOT convention: bin 0 - underflow, nbins+1 - overflow bin
-for i in range(1, nbins+1):
-    for info_entry in all_histoInfos:
+for info_entry in all_histoInfos:
+    nbins = info_entry.th2.GetXaxis().GetNbins()
+    for i in range(1, nbins+1):
         totalEvents = info_entry.th2.GetEntries()
         tmpHist = info_entry.th2.ProjectionY("py",i,i)
+        if("twoStrip" in info_entry.outHistoName): #NewChange - results in matching std. dev and gauss sigma values
+            tmpHist.GetXaxis().SetRangeUser(-0.3,0.3)
```

Comment:
Do we want this range to be always 0.3 for ALL sensors or only pads, for instance?

Reply:
Great! I'll do as you suggested in order to be compatible with future sensors, just in case!